apps/dapp/src/constants/env/local.tsx

```diff
@@ -1,12 +1,36 @@
 import { Environment } from './types';
 
 const env: Environment = {
+  alchemyId: 'VvVv_fBIiRSaTQzL9RQNybD5FSNvtK5c',
+  backendUrl: 'http://localhost:3001',
+  contracts: {
+    exitQueue: '0x0165878A594ca255338adfa4d48449f69242Eb8F',
+    faith: '0xBEc49fA140aCaA83533fB00A2BB19bDdd0290f25',
+    farmingWallet: '',
+    fei: '0x8A791620dd6260079BF849Dc5567aDC3F2FdC318',
+    frax: '0x2279B7A0a67DB372996a5FaB50D91eAA73d2eBe6',
+    frax3CrvFarming: '',
+    frax3CrvFarmingRewards: '',
```

Comment:
What's are these used for?

Reply:
currently pulled into the metrics service to display the treasury. I think this work is migrated to the subgraph though, we can return to the metrics service cleanup in a future issue